src/lib/extract/extractBrush.ts

```diff
@@ -40,6 +40,16 @@ export default function extractBrush(color?: Color) {
     return int32ARGBColor;
   }
 
+  // iOS PlatformColor
+  if ('semantic' in color) {
+    return [0, color];
+  }
+
+  // Android PlatformColor
+  if ('resource_paths' in color) {
+    return [0, color];
+  }
```

Comment:
At least on iOS, "semantic" isn't enough, because a color might be a DynamicColor instead. Could we instead do something like a type check? `if (typeof color === ColorValue) ....`

Reply:
`DynamicColorIOS` isn't supported in this PR, and I don't really see that API surviving long term as it's iOS only.

Reply:
Surviving in react-native, or surviving in this repo? I don't think it's going anywhere as long as PlatformColor exists. We also have a react-native-macos equivalent `DynacmicColorMacOS` that won't go anywhere as long as DynamicColor exists.

Reply:
I meant in `react-native`. `DynamicColorIOS` predates `PlaformColor`, and while it's not able to fully replace it just yet, I don't see that API as useful without cross platform support. I guess is possible to add however.